### DISCUSSION TITLE
Request to be added as maintainer/admin of tgenv organisation

### DISCUSSION THREAD

**Discussion by iokiwi** [tgenv/tgenv#20] [General]
Hey @weyderfs,

Thanks for making the org and fork. I'm stoked with the progress we have made so far our fork is a healthy little project again.

A critical part of my suggestion to create a tgenv org was to spread the bus factor of access/ownership of the account managing the repository so we can keep it alive even if one creator goes AWOL.

As such I wondered if I could please be added as a secondary owner/maintainer/admin of the tgenv organisation too.

**Comment by weyderfs** [comment]
@iokiwi how was going?

First, sorry for time to reply to you here, it was Carnival Holiday here in Brazil e I'm back to work this week. I've invited you to be Admin of our org too.

**Reply by weyderfs** [reply]
@iokiwi Are you received the Org. invitation?

**Comment by weyderfs** [comment]
I'm closing this topic because it was solved, I've promoted @iokiwi like a Admin of tgenv org.
